adoc/chapters/programming_interface.adoc

```diff
@@ -6895,6 +6896,30 @@ accessor_ptr<IsDecorated> get_multi_ptr() const noexcept
 
 This function may only be called from within a <<command>>.
 
+a@
+[source]
+----
+accessor &operator=(const value_type &other) const
```

Comment:
I think this needs to return a `const accessor &` because the member function is `const`.